webapp/src/types/types.ts

```diff
@@ -111,3 +111,9 @@ export type CallsUserPreferences = {
 export const CallsUserPreferencesDefault = {
     joinSoundParticipantsThreshold: 8,
 };
+
+export type EmojiData = {
+    id: string;
```

Comment:
Should this be `name`? I can change it in the server instead, 0/5

Reply:
Oops yes, let me fix this.